sdcm/cluster.py

```diff
@@ -1179,15 +1177,13 @@ def wait_manager_agent_up(self, verbose=True, timeout=180):
 
     def is_manager_server_up(self, port=None):
         port = port if port else self.MANAGER_SERVER_PORT
-        if self.is_port_used(port=port, service_name="scylla-manager"):
-            # When the manager has started,
-            # it should answer an http request of https://127.0.0.1:5080/ping with status code 204
-            curl_output = self.remoter.run(
-                f'''curl --write-out "%{{http_code}}\n" --silent --output /dev/null "http://127.0.0.1:{port}/ping"''',
-                ignore_status=True)
-            http_status_code = int(curl_output.stdout.strip())
-            return http_status_code == 204
-        return False
+        # When the manager has started,
+        # it should answer an http request of https://127.0.0.1:5080/ping with status code 204
+        curl_output = self.remoter.run(
+            f'''curl --write-out "%{{http_code}}\n" --silent --output /dev/null "http://127.0.0.1:{port}/ping"''',
+            verbose=True, ignore_status=True)
+        http_status_code = int(curl_output.stdout.strip())
+        return http_status_code == 204
```

Comment:
same here as above

Reply:
wait_for swallows the excepion, and in any case curl will not fail, it will just return 000

Reply:
here time out is handle in less clear way, since curl error code on that case would be weird.
but if it's keep failing the status code would be 204, but some like like 000 in case of timeout. it's not that clear anyhow, since the exception would be timeout, 
I might change here the return to `assert http_status_code == 204, f"{http_status_code} != 204"`, and then the exception would be move clear